wiki/Skinning/Interface/en.md

```diff
@@ -98,364 +123,485 @@ Since the button has three parts, it is best to keep the heights the same!
 
-**Note:**
+Notes:
 
-- This mod is only used for [osu!standard](/wiki/osu!standard).
+- This is an [osu!standard](/wiki/osu!standard)-specific mod.
 
 ---
 
-![](img/selection-mod-suddendeath.png "selection-mod-suddendeath.png")
+`selection-mod-suddendeath.png`
+
+![](img/selection-mod-suddendeath.png)
+
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Normal | Centre | 64x64 |
 
 ---
 
-![](img/selection-mod-target.png "selection-mod-target.png")
+`selection-mod-target.png`
 
-- This mod is only available for the cuttingedge stream only.
-- This mod is only used for [osu!standard](/wiki/osu!standard).
+![](img/selection-mod-target.png)
 
-## Offset Wizard
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Normal | Centre | 64x64 |
 
-_See also: [Offset Wizard](/wiki/Offset_Wizard)._
+- This mod is available in the cuttingedge stream only.
+- This is an [osu!standard](/wiki/osu!standard)-specific mod.
+
+## Offset wizard
+
+*See also: [Offset Wizard](/wiki/Offset_Wizard).*
 
 ---
 
-![](img/options-offset-tick.png "options-offset-tick.png")
+`options-offset-tick.png`
+
+![](img/options-offset-tick.png)
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![No][false]      |
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![No][false] | Multiplicative | Centre | - |
 
-**Note:**
+Notes:
 
-- This element is stretched to fit screen height.
+- Tinting varies by tick state.
 
 ## Playfield
 
-![](img/play-skip.png "play-skip.png")
+`play-skip.png`
+
+![](img/play-skip.png)
+
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![Yes][true] | ![Yes][true] | Multiplicative | BottomRight | - |
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![Yes][true] | ![Yes][true]      |
 
-**Note:**
+Notes:
 
-- Animation name: `play-skip-#.png`; replace `#` with indexed count (start frame count at zero).
+- Animation name: `play-skip-{n}.png`
 
 ---
 
-![](img/play-unranked.png "play-unranked.png")
+`play-unranked.png`
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
+![](img/play-unranked.png)
 
-**Note:**
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![Yes][true] | Multiplicative | Centre | - |
 
-- Shown when using mods that disable score submission.
+Notes:
+
+- This is shown when using mods that disable score submission or when playing a [pending](/wiki/pending) or [graveyarded](/wiki/graveyarded) beatmap.
```

Comment:
It doesn't depend on the status of a beatmap for it to show up. It solely depends on the mods.